board/fischertechnik/TXT/initramfs/repartition-sd-card

```diff
@@ -1,31 +1,6 @@
 #!/bin/busybox sh
 mount -t vfat /dev/mmcblk1p1 /media/sdcard/boot -o dmask=077,fmask=177,sync
```

Comment:
A suggestion for this file: It is very important that errors are noticed as it is hard to find bugs due to a wrong installation. I would suggest to fail whenever _any_ of the below commands return with a non-zero state. In bash, a line like `set -e` does that. I don't know if `#!/bin/busybox sh -e` in line 1 would do that.

Reply:
New issue: https://github.com/ftCommunity/ftcommunity-TXT/issues/227